sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/ece-response-integration.scheduler.ts

```diff
@@ -18,40 +21,34 @@ export class ECEResponseIntegrationScheduler extends BaseScheduler<void> {
     super(schedulerQueue, queueService);
   }
 
-  /**
-   * To be removed once the method {@link process} is implemented.
-   * This method "hides" the {@link Process} decorator from the base class.
-   */
-  async processQueue(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
-  /**
-   * When implemented in a derived class, process the queue job.
-   * To be implemented.
-   */
-  protected async process(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
   /**
    * Process all the institution ECE responses and update the enrolment accordingly.
-   * @params job details.
+   * @param _job process job.
+   * @param processSummary process summary for logging.
    * @returns Processing result log.
    */
-  @Process()
-  async processECEResponse(
-    job: Job<void>,
-  ): Promise<QueueProcessSummaryResult[]> {
-    this.logger.log(
-      `Processing ECE response integration job ${job.id} of type ${job.name}.`,
-    );
-    this.logger.log("Processing ECE response files ...");
+  protected async process(
+    _job: Job<void>,
+    processSummary: ProcessSummary,
+  ): Promise<string[] | string> {
     const processingResult = await this.eceResponseProcessingService.process();
-    this.logger.log("Processing ECE response files completed.");
-    return processingResult;
+    processingResult.forEach((result) => {
+      // Convert each ECE response processing result to a process summary for a location.
+      const locationResultSummary = new ProcessSummary();
+      processSummary.children(locationResultSummary);
+      result.summary.forEach((info) => locationResultSummary.info(info));
+      result.warnings.forEach((error) => locationResultSummary.warn(error));
+      result.errors.forEach((error) => locationResultSummary.error(error));
```

Comment:
Is there a particular reason why we did not use the process summary inside the processing service here?

Reply:
The service is heavily relying in the current summary and replacing it by the `ProcessSummary` would demand way more refactor, which I considered completely outside the target of this effort.

Reply:
Makes sense. Can we have a ticket created to not lose the track of that effort?